ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html

```diff
@@ -19,143 +19,152 @@
 {% set license = h.ontario_theme_get_license(pkg.license_id) %}
 {% set is_admin = h.check_access('package_update', {'id':pkg.id }) %}
 {% set is_open = (pkg.access_level == 'open') %}
```

Comment:
`is_open` is defined globally here but is not available to the blocks, so can be removed here.